3.3.c

```diff
@@ -0,0 +1,76 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+#include <stdbool.h>
+#define arraySize 15
+#define countArraySize 30

```

Comment:
```suggestion
#define ARRAY_SIZE 15
#define COUMT_ARRAY _SIZE 30
```
Вот принято оформлять имена define определений.